src/taglib.ml

```diff
@@ -276,9 +276,9 @@ struct
       parse_tag (grab_tag t) h ;
       t
    
-    external render : tag -> string = "caml_taglib_id3v2_render"
+    external render : tag -> bytes = "caml_taglib_id3v2_render"
 
-    let render t = render (grab_tag t)
+    let render t = Bytes.to_string (render (grab_tag t))
```

Comment:
Since you have full ownership of the bytes at this point, this is a good use-case for `unsafe_to_string`:

```suggestion
    let render t = Bytes.unsafe_to_string (render (grab_tag t))
```